ui/app/components/secure-variable-form.js

```diff
@@ -1,8 +1,25 @@
+// @ts-check
+
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { A } from '@ember/array';
 
 export default class SecureVariableFormComponent extends Component {
-  keyValues = [{ key: '', value: '' }];
+  path = '';
+  keyValues = A([{ key: '', value: '' }]);
+
+  @tracked
+  hideValues = true;
```

Comment:
`nit` `non-blocking`: When naming boolean parameters I normally like to prefix with `is` `should` and `are`.

e.g.

`hideValues` --> `areValuesHidden`